tbdex/rfq/rfq_test.go

```diff
@@ -81,7 +81,117 @@ func TestRFQ_Sign(t *testing.T) {
 
 func TestRFQ_UnmarshalJSON(t *testing.T) {
 	pfiDID, _ := didjwk.Create()
+	walletDID, _ := didjwk.Create()
+	offeringID, _ := typeid.WithPrefix(offering.Kind)
+
+	r, _ := rfq.Create(
+		walletDID.URI,
+		pfiDID.URI,
+		offeringID.String(),
+		rfq.Payin("100", "STORED_BALANCE"),
+		rfq.Payout("BANK_ACCOUNT"),
+	)
+
+	_ = r.Sign(walletDID)
```

Comment:
do we need the `_ =` in order for linter to not complain?

Reply:
sign returns an error, but i'm not checking it as that functionality was tested above.